firebase-firestore/api.txt

```diff
@@ -19,6 +19,19 @@ package com.google.firebase {
 
 package com.google.firebase.firestore {
 
+  public final class AggregateQuery {
```

Comment:
nit: Make AggregateQuery non-final, to be consistent with Query and Collection and AggregateQuerySnapshot.